includes/Core/Assets/Assets.php

```diff
@@ -533,9 +550,10 @@ private function get_inline_base_data() {
 			'homeURL'          => home_url(),
 			'referenceSiteURL' => esc_url_raw( $site_url ),
 			'userIDHash'       => md5( $site_url . $current_user->ID ),
-			'adminRoot'        => esc_url_raw( get_admin_url() . 'admin.php' ),
-			'assetsRoot'       => esc_url_raw( $this->context->url( 'dist/assets/' ) ),
+			'adminURL'         => esc_url_raw( get_admin_url() . 'admin.php' ),
+			'assetsURL'        => esc_url_raw( $this->context->url( 'dist/assets/' ) ),
```

Comment:
Also doesn't appear to be used 🤔

Reply:
Yes, but these should remain here, they are moved from the old `googlesitekit` global.